config/formats.js

```diff
@@ -457,7 +457,7 @@ let Formats = [
 	},
 	{
 		name: "[Gen 7] Reversed",
-		desc: `Every Pok&eacute;mon has its Atk and Sp. Atk, as well as its Def and Sp. Def, swap.`,
+		desc: `Every Pok&eacute;mon has its Atk and Sp. Atk, as well as its Def and Sp. Def, swapped.`,
```

Comment:
The way it was originally was fine. However, can you add "stats" after Sp. Atk and Sp. Def, and add "base" before Atk and Def?

Reply:
Gotcha